tests/integration/test_logging.py

```diff
@@ -14,4 +14,3 @@ def test_deactivation_of_logger(engine, caplog):
     assert len(caplog.records) == 0
 
     logger.disabled = False
```

Comment:
```suggestion

```
I'd say you don't need this either. And if you do, it'd mean there is something wrong in the testing suite.